supertokens_python/recipe/thirdpartyemailpassword/recipe.py

```diff
@@ -109,6 +113,7 @@ def apis_override_email_password(_: EmailPasswordAPIInterface) -> EmailPasswordA
             self.email_password_recipe = email_password_recipe
         else:
             self.email_password_recipe = EmailPasswordRecipe(recipe_id, app_info,
+                                                             EmailPasswordIngredients(None),
```

Comment:
You should not pass none to this

Reply:
It's fine. It's passed in the PR for tpep https://github.com/supertokens/supertokens-python/pull/163/files#diff-1d423b5d85fc2ebb83925be074e6758f869042c73d9f9a8ac33b96d38757b71eR146